src/test/java/org/folio/edge/sip2/api/support/BaseTest.java

```diff
@@ -4,6 +4,7 @@
 import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.when;
 
+import io.vertx.core.AsyncResult;
```

Comment:
How is this import used? I don't see it in the new code.

Reply:
Weird, I don't have it in my local copy. Thought I had pushed everything up already.